docs/guides/cloud/introduction.mdx

```diff
@@ -73,17 +75,18 @@ the latest results across the team, and identify areas of concern.
 
 ### Run tests in parallel, in priority order, or not at all
 
-With our [Smart Orchestration](/guides/cloud/smart-orchestration) features, you
-can run tests across a swarm of machines simultaneously while Cypress Cloud
-coordinates runners and
-[balances test loads](/guides/guides/parallelization#Balance-strategy) - no
-setup required! You can prioritize recently failed specs with
-[Spec Prioritization](/guides/cloud/smart-orchestration#Spec-Prioritization) to
+With our
+[Smart Orchestration](/guides/cloud/smart-orchestration/parallelization)
```

Comment:
I think removing this link would be better than dropping into parallelization. If there is a landing page some day, we can add this is.